securing/part3.html

```diff
@@ -185,7 +185,7 @@
 
 	<p>Cross-site scripting (XSS) vulnerabilities make it possible for the user to include malicious content to a site that will then be executed on the machine of another user. The malicious content may be permanently stored on the web application (e.g. in a database if the input and output is not sanitized), or it may be included temporarily (e.g. as a part of a query parameter).</p>
 
-	<p>Such attacks make it possible to -- among other things -- hijack users' sessions and as a consequense impersonate another user. For example, the cookies that are related to the current document may be accessible through the variable <code>document.cookie</code>, and sent to the server of a malicious user through a simple XMLHttpRequest.</p>
+	<p>Such attacks make it possible to -- among other things -- hijack users' sessions and as a consequance impersonate another user. For example, the cookies that are related to the current document may be accessible through the variable <code>document.cookie</code>, and sent to the server of a malicious user through a simple XMLHttpRequest.</p>
```

Comment:
thanks, right!

"**consequence**"   should be  under the my edition;  

mistake under the "fixing" mistake. :(

---
I some kind of totally beginner with most of "development" things.
Do I able to re-fix it under the "this pull request" directly?

If yes - this is provided by "change/edit"-picture around this changes-description?